package.json

```diff
@@ -107,8 +110,8 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production gulp build",
     "lint": "eslint .",
-    "checkformatting": "prettier --check '**/*.{js,scss,d.ts}'",
-    "format": "prettier --list-different --write '**/*.{js,scss,d.ts}'",
+    "checkformatting": "prettier --check '**/*.{js,scss,ts,tsx,d.ts}'",
+    "format": "prettier --list-different --write '**/*.{js,scss,ts,tsx,d.ts}'",
```

Comment:
The .tsx extension is required for TypeScript files that contain JSX syntax. This is needed to resolve a conflict with some historical TS syntax which conflicted with JSX.